Megaparsec/Errors/Bundle.lean

```diff
@@ -1,16 +1,16 @@
 import Megaparsec.Stream
 import Megaparsec.Errors.StreamErrors
 import Megaparsec.ParserState
-import Megaparsec.NEList
+import YatimaStdLib
 
 namespace Bundle
 
 structure ParseErrorBundle [stream : Stream.Stream S] {E : Type} where
-  errors : NEList.NEList (@StreamErrors.ParseError S E stream)
+  errors : NEList (@StreamErrors.ParseError S E stream)
   posState : ParserState.PosState S
 
 def toBundle {S : Type} [stream : Stream.Stream S] {E : Type}
-             (s : ParserState.State S E) (errs : NEList.NEList (@StreamErrors.ParseError S E stream))
+             (s : ParserState.State S E) (errs : NEList (@StreamErrors.ParseError S E stream))
```

Comment:
Perhaps we want to show that we can close this now: https://github.com/yatima-inc/Megaparsec.lean/issues/18 ?

Reply:
What exactly?